src/Response/GetAllPostCodesResponse.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\SpeedyBundle\Response;
+
+use Answear\SpeedyBundle\Response\Struct\PostCode;
+use Answear\SpeedyBundle\Response\Struct\PostCodeCollection;
+use Symfony\Component\Serializer\Encoder\CsvEncoder;
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+class GetAllPostCodesResponse
+{
+    public PostCodeCollection $postCodes;
+
+    public function __construct(PostCodeCollection $postCodes)
+    {
+        $this->postCodes = $postCodes;
+    }
+
+    public function getPostCodes(): PostCodeCollection
+    {
+        return $this->postCodes;
+    }
+
+    public static function fromCsv(string $csv): self
+    {
+        $serializer = new Serializer([new ObjectNormalizer(), new ArrayDenormalizer()], [new CsvEncoder()]);
+
+        $postCodes = $serializer->deserialize($csv, PostCode::class . '[]', CsvEncoder::FORMAT);
```

Comment:
this is simple data,
maybe you can do this faster manually

Reply:
done